### DISCUSSION TITLE
how do you putt auto suggestions on

### DISCUSSION THREAD

**Discussion by troopy95** [bridge-core/bridge.#12] [Q&A] [answered]

**Comment by solvedDev** [comment, marked as answer]
Auto-suggestions are on automatically.

In order to use them, "bridge." must know the file type of the .json file you've opened though (entity, item, loot table, etc.). The type of a file is determined by looking at the file path of it. This means if you want entity auto-completions, the file has to be in an "entities" folder like in the vanilla BP.
